plugins/performance-lab/includes/admin/load.php

```diff
@@ -230,6 +227,24 @@ function perflab_enqueue_features_page_scripts(): void {
 
 	// Enqueue the a11y script.
 	wp_enqueue_script( 'wp-a11y' );
+
+	// Enqueue plugin activate AJAX script and localize script data.
+	wp_enqueue_script(
+		'perflab-plugin-activate-ajax',
+		plugins_url( 'includes/admin/plugin-activate-ajax.js', PERFLAB_MAIN_FILE ),
+		array( 'wp-i18n', 'wp-a11y' ),
```

Comment:
As noted in a comment below, this can use `wp-api-fetch` instead if the REST API is used.

```suggestion
		array( 'wp-i18n', 'wp-a11y', 'wp-api-fetch' ),
```